src/OpenTelemetry.AutoInstrumentation/Configuration/PluginsConfigurationHelper.cs

```diff
@@ -49,4 +60,16 @@ private static TracerProviderBuilder InvokePlugin(this TracerProviderBuilder bui
         var result = mi.Invoke(obj, new object[] { builder });
         return (TracerProviderBuilder)result;
     }
+
+    private static MeterProviderBuilder InvokePlugin(this MeterProviderBuilder builder, string pluginAssemblyQualifiedName)
+    {
+        // get the type and method
+        var t = Type.GetType(pluginAssemblyQualifiedName);
+        var mi = t.GetMethod("ConfigureMeterProvider", new Type[] { typeof(MeterProviderBuilder) });
```

Comment:
Check for `null` return, see the symmetric method for trace builder.

Reply:
Fixed fd000743771d05c0e58749ce1bd095c9828a70f2